### ISSUE TITLE
Create Cospend Project

### ISSUE THREAD

**Issue by edwardscaleb9** [mayflower/PayForMe#23]
Would be awesome if we could create a new Cospend project from the app.

**Comment by Hustenbonbon** [comment]
Yes, but unfortunately this is quite tricky, as you need to also authenticate to nextcloud for that and so on. Definitely something we aim for, but might take some time
